### ISSUE TITLE
Help button error.

### ISSUE THREAD

**Issue by Aniket21mathur** [sugarlabs/turtleart-activity#66]
On clicking the help button the conosle shows the following error 
```
  File "/home/guest/turtleart-activity/TurtleArt/util/helpbutton.py", line 76, in __help_button_clicked_cb
    self._palette.popup(immediate=True, state=1)
TypeError: popup() got an unexpected keyword argument 'state'
```

**Comment by quozl** [comment]
This is a common mistake during porting from GTK+ 2 to GTK+ 3 of the Sugar Toolkit, as the API changed.  Perhaps inadequate testing.  You might check the API.
* https://developer.sugarlabs.org/sugar3/
* https://developer.sugarlabs.org/sugar3/sugar3.graphics.palette.html
* https://github.com/sugarlabs/sugar-toolkit-gtk3/blob/master/src/sugar3/graphics/palette.py

**Comment by chimosky** [comment]
This has been fixed in #63

**Comment by quozl** [comment]
b9fe9194c4daabdcbc4ea36ad8ac80fae3497c75
